vrtool_docs/Gebruikershandleiding/Preprocessing/Vakindeling.rst

```diff
@@ -0,0 +1,116 @@
+Vakindeling
+===========
+
+De basis van een veiligheidsrendementberekening is één vakindeling die
+voor alle faalmechanismen gebruikt wordt. Als eerste wordt een
+invoerbestand gevuld, waarna de workflow voor het maken van de
+vakindeling kan worden gedraaid.
+
+Structuur van het invoerbestand van de vakindeling
+-----------------------------------------------
+
+De basis voor het genereren van de vakindeling is het invoerbestand
+``Vakindeling.csv``. Dit bestand is, ervan uitgaande dat de standaardinstallatieinstructies gevolgd zijn terug te vinden in:
+``C:\Veiligheidsrendement\.env\Lib\site-packages\preprocessing\default_files``.Doel van deze workflow is een ``geojson`` bestand te creeren waarin alle vakken geografisch zijn gerepresenteerd. Dit bestand is ook invoer voor andere workflows. Dit bestand wordt in de mappenstructuur weggeschreven in ``intermediate_files\vakindeling\
+
+Het invoerbestand ``Vakindeling.csv`` heeft de volgende kolommen die
+ingevuld moeten worden:
+
+.. csv-table:: Kolommen in invoerbestand vakindeling
+  :file: tables/vakindeling_kolommen.csv
+  :widths: 15, 15, 50
+  :header-rows: 1
+
+.. topic:: Aandachtspunten
+
+  Let op: bij het opgeven van een koppeling in de kolommen ``piping``, ``stabiliteit`` etc. hoeven deze nog niet te zijn ingevuld voor deze workflow. Echter, bij het maken van de database moet het gegenereerde ``geojson`` bestand wel alle benodigde informatie bevatten.
+
+Het vullen van het invoerbestand
+-------------------------------
+
+In onderstaande figuur is met een voorbeeld voor stabiliteit
+geillustreerd hoe de koppeling tussen doorsnedes en de vakindeling moet
+worden ingevoerd. Merk op dat het mogelijk is voor meerdere vakken
+dezelfde doorsnede te hanteren (een voorbeeld in de figuur is de
+dikgedrukte doorsnede ``ET_VOLDOET``). Deze hoeft dan slechts 1x genoemd
+te worden in het STBI invoerbestand, maar kan bij meerdere vakken worden
+gebruikt. 
+
+.. image:: img/Filling_Vakindeling_stbi.PNG
+    :alt: Voorbeeld van het vullen van het invoerbestand van de vakindeling
+    :align: center
+
+Belangrijk bij het genereren van de vakindeling zijn met name de
```

Comment:
Dit staat ook onderaan bij trajectlengte, toch? Of gaat dit over als het begin van vak Y niet overeenkomt van het einde van vak Y-1?

Reply:
Ja, daar is het wat uitvoeriger beschreven. Hier liepen nogal wat gebruikers tegenaan, maar misschien is het te dubbelop.